components/productions/index/productions.vue

```diff
@@ -56,13 +57,17 @@
 </script>
 
 <style lang="scss">
-  .table-productions {
+  .list-productions {
```

Comment:
I would try to avoid prefixing this with the semantic element, .productions-container maybe?

Reply:
Or make the `style` `scoped` and just call it `.productions`?

Reply:
@bertramakers If you make the component scoped, styling from parent components will be excluded. Currently the production component does need certain styling from higher up.

@geoffreydhuyvetters productions-container already exists in index, but maybe I can rename the one in index

Reply:
@simon-debruijn  Why would the parent component, in this case, need to style the child component? Wouldn't it be better to make sure each component has its own styles? Global styles like Bootstrap etc keep working even if you add scoped CSS. You can even add a mix of scoped and non-scoped CSS to a component (https://vue-loader.vuejs.org/guide/scoped-css.html#mixing-local-and-global-styles), but I think the default should be to use `scoped` as much as possible to avoid leaking CSS throughout all our components.

Reply:
Will be implemented in next PR